Server/index.js

```diff
@@ -93,6 +93,16 @@ const startApp = async () => {
 
   await connectDbAndRunServer(app, db);
   const jobQueue = await JobQueue.createJobQueue(db);
+
+  const cleanup = async () => {
```

Comment:
Maybe a try-catch here and logging of errors just in case any errors occur?

Reply:
Good call, that's worth doing!  I will add that to the next PR rather than reopen this one :+1: